src/components/Table/Table.jsx

```diff
@@ -89,9 +89,9 @@ const TableCreator = ({
               }
 
               let fieldEl = null;
-              const bars = relativeBars || percentBars;
+              const bars = relativeBars || percentBars || percentBarsWithValue;
               if (bars) {
-                const altValue = typeof bars === 'function' ? bars(row) : null;
+                const altValue = typeof bars === 'function' && percentBarsWithValue ? bars(row) : null;
```

Comment:
Can't hurt to leave it?  It enforces that percentBarsWithValue is used if we want to show an alt value